docs/Common/Structs/index.md

```diff
@@ -752,6 +752,10 @@
 |:---|:---|:--------|:---------|:----------|
 |type|[Common.FuelType](../enums/#fueltype)|false|||
 |range|Float|false|minvalue: 0<br>maxvalue: 10000|The estimate range in KM the vehicle can travel based on fuel level and consumption|
+|level|Float|false|minvalue: 0<br>maxvalue: 10000|The relative remaining capacity of this fuel type (percentage)|
```

Comment:
minvalue: -6

Reply:
Updated in 86be035deb4426cac61bd292dae37c3be8ae1f64